lib/entice/logic/MapInstance.ex

```diff
@@ -0,0 +1,52 @@
+defmodule Entice.Logic.MapInstance do
+  alias Entice.Entity
+  alias Entice.Logic.MapInstance
+
+
+  defstruct(players: %{},
```

Comment:
lets put the fields all underneath defstruct here...
also, below you're using players like a list, here you declare a map :)